docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -219,7 +219,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Notice how `session_id` and `session_start` do not have any semantic tags currently associated to it.\n",
+    "If a column contains semantic tags, they will appear on the right side of a semicolon in the plot above. Notice how `session_id` and `session_start` do not have any semantic tags currently associated to it.\n",
```

Comment:
grammar nitpick at the end of this sentence - "do not have any semantic tags currently associated to it" the "it" is multiple columns, so it should refer to them:

> Notice how the columns `session_id` and `session_start` do not have any semantic tags currently associated with them.